client/me/purchases/manage-purchase/auto-renew-toggle/index.jsx

```diff
@@ -139,16 +142,27 @@ class AutoRenewToggle extends Component {
 		return this.props.isEnabled;
 	}
 
+	renderTextStatus() {
+		const { translate, isEnabled } = this.props;
+
+		return isEnabled ? translate( 'Auto-renew (on)' ) : translate( 'Auto-renew (off)' );
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **21** times:
`translate( 'Auto-renew is ON' )` *ES Score: 9*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/domain-status-autorenew-toggle)